static/js/src/modules/feedback.js

```diff
@@ -0,0 +1,66 @@
+const { el, mount, textContent, list, setChildren, setStyle, setAttr } = redom
+
+let pageTitle = document.querySelector(".blog-post-title")
+
+let title = `Potential Missing Documentation: ${pageTitle.textContent}`
```

Comment:
I suggest changing the title to "Feedback"

Reply:
I like "Potential Missing Documentation:" because we want the user to be specific i.e. write down what they think is missing so we can either:
- find it and help them or 
- create it and help them.
We want to avoid vague "Feedback" that we can not provide a concrete answer to.